### PR TITLE
Remove switch_browser_content, add the checking of URLBar focus images

### PR DESCRIPTION
**Pull request by askeing**
- remove switch_browser_content
- add the checking of URLBar focus images
- check the window is launched before locking window position
- move window to foreground after locking position
- speed up closeBrowser for Windows and Ubuntu
- skip closeBrowser on Mac, using appscript to quit browser faster

Ref: #477

### CODE REVIEW

**Comment by askeing** [comment]
https://docs.google.com/spreadsheets/d/11JKckd0FUvzMo_jsmwgMm5oP_hPZHsK-3h--48TVLc8/edit#gid=0

**Comment by askeing** [comment]
TODO: remove `tests/test_chrome_switchcontentwindow.sikuli` and `tests/test_firefox_switchcontentwindow.sikuli`

File: lib/sikuli.py [hunk 1]
```diff
@@ -34,7 +36,23 @@ def run_sikulix_cmd(self, script_dir_path, args_list=[]):
...
+            elif browser.lower() == 'chrome':
+                appname_list = ['Google Chrome']
+            for appname in appname_list:
+                try:
+                    browser_obj = app(appname)
+                    break
+                except:
+                    pass
+            if browser_obj:
+                browser_obj.quit()
```
Comment (askeing) [inline lib/sikuli.py]: Should put it into `try` after Ln. 48, for reducing condition checking.

**Comment by askeing** [comment]
@MikeLien @ShakoHo r?

**Comment by ShakoHo** [comment]
lgtm r+

**Pull request merged**